.travis.yml

```diff
@@ -0,0 +1,12 @@
+dist: Xenial
+language: node_js
+node_js:
+  - "6"
+before_install:
+  - wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+  - sudo add-apt-repository -y 'deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial main'
+  - sudo apt update
+  - sudo apt-get install -y clang-format-9
```

Comment:
I think you should run ` sudo apt-get install -y clang-format-9` and `npm install -g markdownlint-cli` during the `install` stage.